service-registry-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/externalmodel/ServiceRecord.java

```diff
@@ -25,15 +29,16 @@ public ServiceRecord(String serviceIdentifier, String organisationNumber, String
         this.endPointURL = endPointURL;
     }
 
-    public ServiceRecord(String serviceIdentifier, String organisationNumber, String pemCertificate
-            , String endPointURL, String epostAdresse, String varslingsStatus, String mobilnummer) {
+    private ServiceRecord(String serviceIdentifier, String organisationNumber, String pemCertificate
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00107)